neurokit2/stats/cluster_quality.py

```diff
@@ -86,7 +86,7 @@ def cluster_quality(data, clustering, clusters=None, info=None, n_random=10, **k
 
     # Variance explained
     general["Score_VarianceExplained"] = _cluster_quality_variance(data, clusters)
-    general["Score_GEV"] = _cluster_quality_gev(data, clusters, clustering, **kwargs)
+    general["Score_GEV"], _ = _cluster_quality_gev(data, clusters, clustering, **kwargs)
```

Comment:
we should add individual GEVs in the output of cluster_quality

Reply:
Currently it's returned in the info of microstates_segment(). I'm not sure how should we return it in cluster_quality()

Reply:
ok let's leave it as is for now